x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/edit_policy.js

```diff
@@ -0,0 +1,324 @@
+                            defaultMessage="Name"
+                          />
+                        </span>
+                      </div>
+                    }
+                    titleSize="s"
+                    fullWidth
+                  >
+                    <ErrableFormRow
+                      id={STRUCTURE_POLICY_NAME}
+                      label={intl.formatMessage({
+                        id: 'xpack.indexLifecycleMgmt.editPolicy.policyNameLabel',
+                        defaultMessage: 'Policy name',
+                      })}
+                      errorKey={STRUCTURE_POLICY_NAME}
+                      isShowingErrors={isShowingErrors}
+                      errors={errors}
+                    >
+                      <EuiFieldText
+                        value={selectedPolicyName}
+                        onChange={async e => {
+                          await setSelectedPolicyName(e.target.value);
+                        }}
+                      />
+                    </ErrableFormRow>
+                  </EuiDescribedFormGroup>
+                ) : null}
+              </Fragment>
+              <EuiSpacer />
+              <HotPhase
+                selectedPolicy={selectedPolicy}
+                errors={errors[PHASE_HOT]}
+                isShowingErrors={isShowingErrors && !!findFirstError(errors[PHASE_HOT], false)}
+              />
+              <EuiHorizontalRule className="ilmHrule" />
+              <WarmPhase
+                errors={errors[PHASE_WARM]}
+                showNodeDetailsFlyout={this.showNodeDetailsFlyout}
+                isShowingErrors={isShowingErrors && !!findFirstError(errors[PHASE_WARM], false)}
+              />
+              <EuiHorizontalRule className="ilmHrule" />
+              <ColdPhase
+                errors={errors[PHASE_COLD]}
+                showNodeDetailsFlyout={this.showNodeDetailsFlyout}
+                isShowingErrors={isShowingErrors && !!findFirstError(errors[PHASE_COLD], false)}
+              />
+              <EuiHorizontalRule className="ilmHrule" />
+              <DeletePhase
+                errors={errors[PHASE_DELETE]}
+                isShowingErrors={isShowingErrors && !!findFirstError(errors[PHASE_DELETE], false)}
+              />
+              <EuiHorizontalRule className="ilmHrule" />
+              <EuiFlexGroup justifyContent="spaceBetween">
+                <EuiFlexItem grow={false}>
+                  <EuiFlexGroup>
+                    <EuiFlexItem grow={false}>
+                      <EuiButton
+                        fill
+                        iconType="check"
+                        iconSide="left"
+                        onClick={this.submit}
+                        color="secondary"
+                      >
+                        <FormattedMessage
+                          id="xpack.indexLifecycleMgmt.editPolicy.saveButton"
+                          defaultMessage="Save your policy"
```

Comment:
I think we want to avoid pronouns on action buttons

```suggestion
                          defaultMessage="Save policy"
```